### DISCUSSION TITLE
Videos de las clases

### DISCUSSION THREAD

**Discussion by AristidesGracia** [Launch-X-Latam/MisionFrontEnd#174] [Q&A]
Buen día

Cuando suben los videos de las clases ?

Saludos

**Comment by itsAmauriDev** [comment]
Si te refieres a las grabaciones de las clases las puedes ver aquí https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a solo inicia sesión con tu cuenta de innovacción o también puedes ver las clases nuevamente entrando directamente en los links de las transmisiones.

**Comment by federico42o** [comment]
En el teams pusieron todas los links de las clases
